script.reddit.reader/resources/lib/httpd.py

```diff
@@ -0,0 +1,75 @@
+#file httpd.py
+#http://forum.kodi.tv/showthread.php?tid=204064
+
```

Comment:
It is not used. It is left there as reference in case I need to implement a more user friendly way of doing oauth.